config/default.yml

```diff
@@ -4400,11 +4400,12 @@ Style/NumericLiterals:
   StyleGuide: '#underscores-in-numerics'
   Enabled: true
   VersionAdded: '0.9'
-  VersionChanged: '0.48'
+  VersionChanged: '<<next>>'
```

Comment:
It looks like update is unneeded. Because adding this `AllowedPatterns` configuration does not change the default behavior.

```suggestion
  VersionChanged: '0.48'
```

Reply:
Sure, updated.